generator/internal/genclient/translator/protobuf/protobuf.go

```diff
@@ -182,11 +182,20 @@ func normalizeTypes(in *descriptorpb.FieldDescriptorProto, field *genclient.Fiel
 		// Repeated fields are not optional, they can be empty, but always have
 		// presence.
 		field.Optional = !field.Repeated
+		message, ok := state.MessageByID[field.TypezID]
```

Comment:
nit: may be reduced to `if message, ok := state.MessageByID[field.TypezID]; ok {`

Reply:
Thanks, done.